src/commands/create/contract.ts

```diff
@@ -467,15 +446,15 @@ export default class Contract extends Command {
               royaltyBps, // percentage of royalties in bps
               false, // enableOpenSeaRoyaltyRegistry
               salesConfig,
-              metadataRenderer.address, // metadataRenderer
+              METADATA_RENDERER_ADDRESS,
```

Comment:
Use it as `METADATA_RENDERER_ADDRESS[ENVIRONMENT]` to load specific address based on current environment.

Reply:
done